src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,30 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao abstractDao;
+
+    public CountryServiceImpl(CountryDaoImpl countryDao) {
+        this.abstractDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return abstractDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        Optional<Country> movie = abstractDao.get(id);
```

Comment:
The variable name `movie` is misleading in this context. It should be renamed to something like `country` to reflect that it holds a `Country` object.